ehtim/array.py

```diff
@@ -101,20 +119,45 @@ def obsdata(self, ra, dec, rf, bw, tint, tadv, tstart, tstop, mjd=MJD_DEFAULT, t
 
         """
 
-        obsarr = simobs.make_uvpoints(self, ra, dec, rf, bw,
-                                            tint, tadv, tstart, tstop,
-                                            mjd=mjd, polrep=polrep, tau=tau,
-                                            elevmin=elevmin, elevmax=elevmax,
-                                            timetype=timetype, fix_theta_GMST = fix_theta_GMST)
+        obsarr = simobs.make_uvpoints(
+            self,
+            ra,
+            dec,
+            rf,
+            bw,
+            tint,
+            tadv,
+            tstart,
+            tstop,
+            mjd=mjd,
+            polrep=polrep,
+            tau=tau,
+            elevmin=elevmin,
+            elevmax=elevmax,
+            timetype=timetype,
+            fix_theta_GMST=fix_theta_GMST)
 
         uniquetimes = np.sort(np.unique(obsarr['time']))
-        scans = np.array([[time-0.5*tadv, time+0.5*tadv] for time in uniquetimes]) 
-        source=str(ra) + ":" + str(dec)
-        obs = ehtim.obsdata.Obsdata(ra, dec, rf, bw, obsarr, self.tarr, 
-                                    source=source, mjd=mjd, timetype=timetype, polrep=polrep,
-                                    ampcal=True, phasecal=True, opacitycal=True, 
-                                    dcal=True, frcal=True,
-                                    scantable=scans)
+        scans = np.array([[time - 0.5 * tadv, time + 0.5 * tadv]
+                          for time in uniquetimes])
+        source = str(ra) + ":" + str(dec)
```

Comment:
```suggestion
        source = f"{a}:{dec}"
```
I think this is more readable